src/HTTPBackend.js

```diff
@@ -30,20 +30,46 @@ export default function(options) {
 
 	function parseResponse(request, response) {
 		return new Promise(function(resolve, reject) {
-			response.json().then((json) => {
-
-				let headers = { ...request.header };
+			let headers = { ...request.header };
+			delete headers.resource;
+			delete headers.parameters;
+			delete headers.queryParameters;
+
+			response.text().then(text => {
```

Comment:
I like that, though I will probably still potentially want the error message to resolve back to the original requester